lora-codec-lansitec/readme.md

```diff
@@ -16,7 +16,7 @@ LPWAN codec microservice automatically exposes one REST endpoint using the path
 
```

Comment:
@BeateRixen Can you review the readme file present in the PR? The readme file illustrates the steps the developer needs to perform to implement a sample microservice

Reply:
@MWindel Can you please review this readme file? @ashhSAG93 Can Marlene directly do the required changes in the source file?

Reply:
@BeateRixen. Yes. @MWindel can do the changes on the readme file